packages/storefront-lwc/src/modules/commerce/productDetail/__tests__/productDetail.test.js

```diff
@@ -46,42 +46,6 @@ describe('<commerce-product-detail>', () => {
         ).toMatchSnapshot();
     });
 
-    it('should render carousel', async () => {
```

Comment:
I moved these tests out of product detail and into a separate test for the new product-image-carousel component